package.json

```diff
@@ -18,6 +18,7 @@
   },
   "homepage": "https://github.com/mixpanel/eslint-config",
   "peerDependencies": {
-    "eslint": ">=3.1.0 <4.0.0"
+    "eslint": ">=3.1.0 <4.0.0",
+    "tslint": ">=5.0.0"
```

Comment:
Isn't this a little dangerous, allowing any major version bump indiscriminately?

Reply:
good call, will keep a major version restriction. while we're here, do you see any reason not to bump up peer dependency eslint version to `<5.0.0`? nothing jumping out at me here: https://eslint.org/docs/user-guide/migrating-to-4.0.0#eslint-recommended-changes